src/main/flight/mixer.h

```diff
@@ -97,6 +97,8 @@ typedef struct servoParam_t {
     int16_t max;                            // servo max
     int16_t middle;                         // servo middle
     int8_t rate;                            // range [-100;+100] ; can be used to ajust a rate 0-100% and a direction
+    int8_t minLimit;                        // range [-90;90] ; can be used to adjust a rate 0-100% and a direction
+    int8_t maxLimit;                        // range [-90;90] ; can be used to adjust a rate 0-100% and a direction
```

Comment:
with reference to configurator pull request https://github.com/cleanflight/cleanflight-configurator/pull/193

the comments don't match the usage and the variable names don't match the values!

```
                            'minLimit': data.getInt8(i + 7), //degree -90 90
                            'maxLimit': data.getInt8(i + 8) //degree -90 90
```
perhaps these should be called something like `minAngle` / `maxAngle`